packages/flutter/lib/src/material/button.dart

```diff
@@ -57,6 +58,10 @@ class RawMaterialButton extends StatefulWidget {
   /// If this is set to null, the button will be disabled, see [enabled].
   final VoidCallback onPressed;
 
+  /// Padding to increase the size of the gesture detector which doesn't
+  /// increase the visible material of the button.
+  final EdgeInsets slopPadding;
```

Comment:
(My non-native english skills might be failing me here), but why is this called **slop**Padding?

Reply:
Also, should we document that this actually increases the layout size of the FAB? i.e. without extra work(e.g. using a stack or similar) you can't place things in the slopPadding.

Reply:
I named it this to relate it to hitSlop, but we could just call it outerPadding or externalPadding to be more literal